File: ATV_1_Estruturas_de_selecao/exercicio6.py
def mediaAluno(n1, n2):
    media = (n1 + n2) / 2
    if media >= 7:
        return("Sua media foi de {}, logo esta Aprovado".format(media))
    elif media < 3:
        return("Sua media foi de {}, logo esta Reprovado".format(media))
    else:
        return("Sua media foi de {}, logo esta em Exame".format(media))

File: ATV_1_Estruturas_de_selecao/exercicio6.hs
mediaAluno n1 n2
    |media >= 7 = "Sua media foi de " ++ show(media) ++ ", logo esta Aprovado"
    |media < 3 = "Sua media foi de " ++ show(media) ++ ", logo esta Reprovado"
    |otherwise = "Sua media foi de " ++ show(media) ++ ", logo esta em Exame"
    where
      media = (n1 + n2) / 2
      